styles/about.css

```diff
@@ -0,0 +1,42 @@
+#aboutme {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    list-style: None;
+    justify-content: space-around;
+    padding: 15px;
+}
+  .aboutimage {
+      max-width:300px;
+      height: auto;
+      max-height: 275px !important;
+      margin: 15px;
+      justify-content: baseline;
+  } 
```

Comment:
In the future, try to avoid using `!important`. There is often another way to structure your CSS that doesn't need it and is usually clearer.